docker-compose.yml

```diff
@@ -20,6 +20,8 @@ services:
     container_name: applications-service-api
     environment:
       APPLICATIONS_WEB_BASE_URL: http://forms-web-app:9004
+      BACK_OFFICE_BLOB_STORAGE_DEADLINE_SUBMISSION_URL: http://localhost:10000/
```

Comment:
Might be a silly question, where is this running locally or it is just a placeholder?

Reply:
Good question! This is the default address for Blob Storage when you run [Azurite](https://learn.microsoft.com/en-us/azure/storage/common/storage-use-azurite?tabs=visual-studio)

Reply:
Ah I see. I asked because if its going to be running as a separate docker container we could make use of the docker network configs to reference the url.

Will making it work locally end to end be part of the next ticket?